testing/code/test_source.py

```diff
@@ -417,15 +411,15 @@ def f():
 """
 
     lines = deindent(inspect.getsource(f).splitlines())
-    assert lines == ["def f():", '    c = """while True:', "    pass", '"""']
+    assert lines == ["    def f():", '        c = """while True:', "    pass", '"""']
```

Comment:
this looks pretty much broken now

Reply:
This is the same ad the test above, but in list form, I'm pretty sure this is more correct than before (before it introduced an indentation error)

Reply:
i see what you mean, please make it a indented block with disabled black formatting and a comment that the indent of the last `"""` is the relative ancor

Reply:
I feel I could also delete this test since the above one covers the same behavior and is more expressive :man_shrugging:

Reply:
hmm, true actually